README.md

```diff
@@ -82,61 +97,15 @@ Not all algorithms are equally well-supported on all platforms. In case of quest
   <li>Other Platforms were not tested yet.</li>
 </ul>
 
-### `Development`
-`api`:
-
-1. First, open a Terminal / Command-Line on this project's folder and run:<br/>
-```bash
-$ docker compose up
-```
- - You need to verify that 2 images went up and are running ("curl" & "nginx") using `docker ps`.<br/><br/>
-2. Second, run the app:
-
-```bash
-# development
-$ npm run start
-
-# watch mode
-$ npm run start:dev
-
-# production mode
-$ npm run start:prod
-```
-
-3. Lastly, in order to see any results, you can either:
-- Run Postman with a POST request:
-```bash
-curl --location 'http://localhost:3010/test/' \
---header 'Content-Type: application/json' \
---data '{
-    "algorithm": "kyber512",
-    "iterationsCount": 5
-}'
-```
-- Run the FrontEnd Portal (pqc-quantum-portal), from this repository: <ENTER_GITHUB_PORTAL_URL_HERE>
-
-`portal`:
-
-1. Start a development server.
-    - Working with mock-data server:
-         - In the root folder, run the client:
-            ```bash
-            cd mock-server
-            yarn
-            yarn start:mock:client
-            ```
-         - Inside `mock-server` folder, run `yarn serve` (if the command fails, run `yarn run-server` to start).
-
-    - Working with running backend, from this repository: <ENTER_GITHUB_BACKEND_URL_HERE>.
-        ```bash
-          yarn start:quantum-api
-        ```
-2. Open the browser with `http://localhost:3010/`
 
 # Contributing
 
 Information about how to contribute, can be found [here](CONTRIBUTING.md).
 
+# Releases
+
+Please note, a  Changelog for more details about a  releases will be available soon.
```

Comment:
remove a before releases

suggestion:
Please note, for more details about releases, a Changelog file will be available soon.